src/content/es/fundamentals/codelabs/your-first-pwapp/index.md

```diff
@@ -917,45 +925,45 @@ La consulta de medios `display-mode` permite aplicar estilos dependiendo de cóm
 }
 ```
 
-También puede comprobar el `display-mode` consulta de medios en [JavaScript to see if you're running in standalone](/web/fundamentals/app-install-banners/#detect-mode) .
+También puedes comprobar la media query `display-mode` con [JavaScript para ver si estás ejecutandola independientemente](/web/fundamentals/app-install-banners/#detect-mode) .
 
 ### Bonus: Desinstalando tu PWA
 
-Recuerde, `beforeinstallevent` no se dispara si la aplicación ya está instalada, por lo que durante el desarrollo probablemente querrá instalarla y desinstalarla varias veces para asegurarse de que todo funciona como se esperaba.
+Recuerda, `beforeinstallevent` no se dispara si la aplicación ya está instalada, por lo que durante el desarrollo probablemente querrá instalarla y desinstalarla varias veces para asegurarse de que todo funciona como se esperaba.
```

Comment:
`...querrás...` `...para asegurarte...`  to keep the conjugation in the second person.